apiclient/response_handlers.py

```diff
@@ -33,7 +33,10 @@ class JsonResponseHandler(BaseResponseHandler):
     @staticmethod
     def get_request_data(response: Response) -> JsonType:
         try:
-            response_json = response.json()
+            if response.text == "":
+                response_json = response
```

Comment:
Shouldn't this function return a blank string in this case?

Reply:
I had originally had it returning a blank string, but then switched it to the behavior of RequestResponseHandler, which is to return the response.  Either way works, I just wasn't sure what to be consistent with.

Reply:
I think in this case, if there is no body, then returning `None` would be appropriate, explicitly indicating that there is no content.